pkg/apis/targets/v1alpha1/datadog_types.go

```diff
@@ -48,6 +48,10 @@ type DatadogTargetSpec struct {
 	// DatadogApiKey represents how Datadog credentials should be provided in the secret
 	DatadogAPIKey SecretValueFromSource `json:"apiKey"`
 
+	// DatadogSite controls the site of the Datadog intake API, defaults to `datadoghq.com`
+	// +optional
+	DatadogSite *string `json:"datadogSite,omitempty"`
```

Comment:
CRD element is named `site` vs this one named `datadogSite`
```suggestion
	DatadogSite *string `json:"site,omitempty"`
```

Reply:
Indeed, that was Goland autocomplete 🤦